src/release/breaking-changes/image-provider-load-buffer.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Adding ImageProvider.loadBuffer
+description: ImageProviders must now be implemented using the new loadBuffer
+API instead of the existing load API.
```

Comment:
The description must be one line.

```suggestion
description: ImageProviders must now be implemented using the new loadBuffer API instead of the existing load API.
```